### PR TITLE
JAMES-1789 Refresh list of available mailets/matcher in server and ad…

### PR DESCRIPTION
**Pull request by chibenwa**
…d experimental sections

We need to rationalise the mailet page...

The proposal is to rely on the mailet plugin. I updated the website accordingly.

It is a real problem because no documentation at all was available for ~ 60% of available mailets, including **RemoteDelivery**.

Here are some visuals of the generated page:

m2

m3

m4

### CODE REVIEW

File: src/site/site.xml [hunk 1]
```diff
@@ -161,7 +161,6 @@
                         </item>
                         <item name="Provided Extensions" href="/server/dev-provided.html" collapse="true" >
                             <item name="Mailets" href="/server/dev-provided-mailets.html" />
```
Comment (aduprat) [inline src/site/site.xml]: name="Mailets & Matchers"

File: src/site/xdoc/server/dev-provided.xml [hunk 2]
```diff
@@ -27,9 +27,9 @@
 
 <section name="James Provided Extensions">
 
-  <subsection name="Provided Mailet">
+  <subsection name="Provided Mailet and Matchers">
   
-    <p>See the <a href="dev-provided-mailets.html">provided mailets</a> page for more information.</p>
+    <p>See the <a href="dev-provided-mailets.html">provided mailets and latchers</a> page for more information.</p>
```
Comment (aduprat) [inline src/site/xdoc/server/dev-provided.xml]: Almost that ;)

File: src/site/xdoc/server/feature-mailetcontainer.xml [hunk 3]
```diff
@@ -124,18 +124,17 @@
         <p>James bundles a number of Matchers and Mailets in its distribution.  Descriptions of provided matchers 
         can be found <a href="dev-provided-matchers.html">here</a>, while descriptions of provided mailets can be found 
         <a href="dev-provided-mailets.html">here</a>.</p>
-        
+
+        <p>You can retrieve the full list of mailets and matchers packeged in James <a href="dev-provided-mailets.html"></a></p>
```
Comment (quynhn) [inline src/site/xdoc/server/feature-mailetcontainer.xml]: s/packeged/packaged/

File: src/site/xdoc/server/dev-provided.xml [hunk 4]
```diff
@@ -27,9 +27,9 @@
 
 <section name="James Provided Extensions">
 
-  <subsection name="Provided Mailet">
+  <subsection name="Provided Mailet and Matchers">
```
Comment (quynhn) [inline src/site/xdoc/server/dev-provided.xml]: s/Mailet/Mailets/?

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**